neuralprophet/forecaster.py

```diff
@@ -1082,7 +1082,29 @@ def add_lagged_regressor(self, name, regularization=None, normalize="auto", only
             as_scalar=only_last_value,
         )
         return self
-
+    
+    def add_lagged_regressors(self, names, regularization=None, normalize="auto", only_last_value=False):
```

Comment:
lets merge this function with add_lagged_regressor 
-> if names is list, iterate over it, else just add single regressor

Reply:
that means that we do not create a separate function but add this functionality to the existing add_lagged_regressor method